content/docs/command-reference/queue/logs.md

```diff
@@ -0,0 +1,191 @@
+## queue logs
+
+Show output logs for running and completed tasks in the
+[DVC Experiment](/doc/user-guide/experiment-management/experiments-overview)
+task queue.
+
+## Synopsis
+
+```usage
+usage: dvc queue logs [-h] [-q | -v] [-e <encoding>] [-f] <task>
+
+positional arguments:
+  <task>                Task to show.
+```
+
+## Description
+
+Shows output logs for the specified running or completed experiment task.
+
+By default, this command will show any available log data and then exit. For
+tasks which are still running, the `--follow` option can be used to attach to
+the task and continuously show live log output, until the task has completed.
+
+When using the `--follow` option, it is safe to stop following output using
+`Ctrl+C` (or `SIGINT`). This will only cause the logs command to exit, and the
+experiment task will continue to be run in the background.
+
+## Options
+
+- `-e <encoding>`, `--encoding <encoding>` - text encoding for log output.
+  Defaults to the system locale encoding.
+
+  > ⚠️ Note that this option is used to specify the encoding of the experiment
+  > task output (i.e. the output of pipeline stage commands), which may not
+  > always match the encoding of your system terminal.
+
+- `-f`, `--follow` - attach to task and follow additional live output. Only
+  applicable of the task is still running.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+> This is based on our [Get Started](/doc/start/experiments), where you can find
+> the actual source code.
+
+<details>
+
+### Expand to prepare the example ML project
+
+Clone the DVC repo and download the data it <abbr>depends</abbr> on:
+
+```dvc
+$ git clone git@github.com:iterative/example-get-started.git
+$ cd example-get-started
+$ dvc pull
+```
+
+Let's also install the Python requirements:
+
+> We **strongly** recommend creating a
+> [virtual environment](https://python.readthedocs.io/en/stable/library/venv.html)
+> first.
+
+```dvc
+$ pip install -r src/requirements.txt
+```
+
+</details>
```

Comment:
This is great but may not be needed. Examples in cmd ref don't need to be reproducible.

Reply:
I just copied this block from the `exp` command ref examples. I wasn't sure if this it's supposed to be standard for all of the exp/queueing related examples

Reply:
@jorgeorpinel WDYT?

Reply:
> copied this block from the exp command ref

There are some old commands that still have these but indeed it's easier to omit them (easier to maintain). No need to make them reproducible.